spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -48,18 +49,15 @@
 import org.apache.spark.sql.types.StructField;
 import org.apache.spark.sql.types.StructType;
 import org.assertj.core.api.Assertions;
+import org.assertj.core.api.Assumptions;
 import org.joda.time.DateTime;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Assume;
-import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
-import org.junit.runners.Parameterized.Parameters;
-
-public class TestAddFilesProcedure extends SparkExtensionsTestBase {
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.io.TempDir;
+
```

Comment:
```suggestion
@ExtendWith(ParameterizedTestExtension.class)
```
technically it's not needed, but it's better to be explicit here so that it's immediately known that this is a parameterized test

Reply:
sure. will add this to other test classes.